.env

```diff
@@ -33,3 +33,5 @@ APP_LOG_SUBJECT="[Wikimedia OCR]"
 APP_IMAGE_HOSTS=upload.wikimedia.org,upload.wikimedia.beta.wmflabs.org
 
 APP_CACHE_TTL=3600 # 1 hour
+
+CLIENT_ID=
```

Comment:
Should be e.g. `APP_TRANSKRIBUS_CLIENT_ID` to make it clearer what it's for and to use the standard `APP` prefix.